.github/workflows/orca.yml

```
@@ -92,6 +92,31 @@ jobs:
           php-version: ${{ matrix.php-version }}
           coverage: xdebug
 
+      - name: Install google-chrome-stable
+        run: |
+          # Remove existing google chrome and install required version.
+          sudo dpkg -r google-chrome-stable
+          CHROME_VERSION_STRING="114.0.5735.198-1"
+          wget --no-verbose -O /tmp/chrome.deb https://dl.google.com/linux/chrome/deb/pool/main/g/google-chrome-stable/google-chrome-stable_114.0.5735.198-1_amd64.deb \
```

I think you forgot to use your new variable here:
```suggestion
          wget --no-verbose -O /tmp/chrome.deb "https://dl.google.com/linux/chrome/deb/pool/main/g/google-chrome-stable/google-chrome-stable_${CHROME_VERSION_STRING}_amd64.deb" \
```